.deploy/preprod.yaml

```diff
@@ -55,10 +55,12 @@ spec:
         - application: familie-klage-frontend
         - application: familie-klage-frontend-lokal
         - application: familie-ef-sak
+        - application: familie-ef-sak-lokal
         - application: familie-prosessering
         - application: familie-prosessering-lokal
     outbound:
       rules:
+        - application: familie-ef-sak-lokal
```

Comment:
Er dette riktig, trengs den? 👀

Reply:
Sak skal kalle på klage (Hent klagebehandlinger) og klage skal kalle på sak (hent vedtak) :) 

Vi ble egentlig litt usikre på om den trengs for lokal kjøring, men?

Reply:
Brukes disse policyene lokalt og når man sjekker token med M$-login-opplegget?

Reply:
Vi sjekker litt mer og rydder.

Reply:
✅